contracts/rateproducer/rateproducer.js

```diff
@@ -1,27 +0,0 @@
-let rate = {
```

Comment:
@kriskoin  lets not delete this file

Reply:
@xavier506  so sorry it was a mistake, fixed !!